systemvm/patches/debian/config/opt/cloud/bin/cs_iptables_save.py

```diff
@@ -2,6 +2,22 @@
 #
 # -*- coding: utf-8 -*-
 #
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I mailed the author of iptables_convert , Johannes Hubertz <johannes@hubertz.de>, @sl0, and he agreed to publish his code under apache license version2. so I am forgiven.
He has a focus on security and python iptables utilities so I will work with him on the best integration into CloudStack of his code.